porch/pkg/git/package.go

```diff
@@ -203,6 +203,10 @@ func (p *gitPackageRevision) GetUpstreamLock() (kptfile.Upstream, kptfile.Upstre
 	return *kf.Upstream, *kf.UpstreamLock, nil
 }
 
+func (p *gitPackageRevision) GetLock() (kptfile.Upstream, kptfile.UpstreamLock, error) {
+	return p.findParent()
```

Comment:
do we still need to keep `findParent`? Seems like an extra layer for no particular reason, suggest just folding the `findParent` code into this method.

Reply:
Removed `findParent`.